tsconfig.json

```diff
@@ -1,30 +1,6 @@
 {
-  "compilerOptions": {
```

Comment:
> I'd rather make this config for base config either cjs or es2015 config and get rid one of them. don't feel we need unusued one.

`src`/`include` in tsconfig.json would be a problem to do it. Our build system use `dist/{cjs,es6}/**/*` as build source, not `src/` directly.

If we change your proposed one, we also need to change the build source dir in other parts. It might be [a chance to fix the above problem](https://github.com/ReactiveX/rxjs/pull/2676#discussion_r122578326), but I'm not sure about that such refactoring would resolve them at once.